tests/phpunit/api/v3/ContactTest.php

```diff
@@ -3026,7 +3026,7 @@ public function testContactCreationPermissions($version) {
     $config = CRM_Core_Config::singleton();
     $config->userPermissionClass->permissions = ['access CiviCRM'];
     $result = $this->callAPIFailure('contact', 'create', $params);
-    $this->assertContains('failed', $result['error_message'], 'lacking permissions should not be enough to create a contact');
```

Comment:
@seamuslee001 can we hold off this PR until the one that I have on this file is merged - since it's a fairly large challenging cleanup on this file & I don't want this to make it stale once it is passing